<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QwtPlot" name="qwtPlot"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QLabel" name="bLabel">
          <property name="text">
           <string>B:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="cDoubleSpinBox"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="dLabel">
          <property name="text">
           <string>D:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="aLabel">
          <property name="text">
           <string>A:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="cLabel">
          <property name="text">
           <string>C:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="aDoubleSpinBox"/>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="bDoubleSpinBox"/>
        </item>
        <item row="1" column="3">
         <widget class="QDoubleSpinBox" name="dDoubleSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="tauLabel">
          <property name="text">
           <string>τ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="tauComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="nLabel">
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="nDoubleSpinBox"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="omegaLabel">
          <property name="text">
           <string>Ω</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="omegaComboBox"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="kLabel">
          <property name="text">
           <string>k</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="kDoubleSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="helpButton">
        <property name="text">
         <string>Справка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
